webapp/src/ts/services/contact-save.service.ts

```diff
@@ -171,13 +174,18 @@ export class ContactSaveService {
   }
 
   save(form, docId, type) {
+    return this.ngZone.runOutsideAngular(() => this._save(form, docId, type));
```

Comment:
I think it'd be safer to inline `_save` here, so it won't be called accidentally and there's no confusion over why there's two of them. Also it saves on duplicating the args...